Documentation~/Diagnostics.md

```diff
@@ -11,8 +11,9 @@ Note that there are different ranges within both Code and Settings diagnostic ID
 - Settings:
   - `PAS0xxx`: Unity settings 
   - `PAS1xxx`: other settings IDs defined in code rather than in the json
-  - `PAT0xxx`: texture related settings
-  - `PAM0xxx`: mesh related settings
+  - `PAA0xxx`: texture related settings
```

Comment:
Probably want a new heading in here, saying "Assets", to differentiate between the PAS and PAA issues

Reply:
Done